frontend/public/components/notification-drawer.tsx

```diff
@@ -108,20 +111,37 @@ const getUpdateNotificationEntries = (
   isLoaded: boolean,
   updateData: ClusterUpdate[],
   isEditable: boolean,
+  newerChannel: string,
+  newerChannelVersion: string,
   toggleNotificationDrawer: () => void,
 ): React.ReactNode[] =>
   isLoaded && !_.isEmpty(updateData) && isEditable
     ? [
         <NotificationEntry
           actionPath="/settings/cluster"
           actionText="Update cluster"
-          key="cluster-udpate"
+          key="cluster-update"
           description={updateData[0].version || 'Unknown'}
           type={NotificationTypes.update}
           title="Cluster update available"
           toggleNotificationDrawer={toggleNotificationDrawer}
           targetPath="/settings/cluster"
         />,
+        newerChannel && (
+          <NotificationEntry
+            actionPath="/settings/cluster?showChannels"
+            actionText="Edit channel"
```

Comment:
Should we change this to 'Update channel' to match the title of the modal dialog that opens?

Reply:
I agree.

Reply:
Done.